### PR TITLE
qfix: Update to latest sdk manually

### PR DESCRIPTION
**Pull request by denis-tingaikin**
Signed-off-by: denis-tingaikin <denis.tingajkin@xored.com>

### CODE REVIEW

File: test/client_endpoint_test.go [hunk 1]
```diff
@@ -136,11 +147,11 @@ func (f *NsmgrTestSuite) TestNSmgrEndpointSendFD() {
 
 	f.registerCrossNSE(ctx, setup, nseRegClient, t)
 
-	cl := client.NewClient(context.Background(),
+	cl := retry.NewClient(client.NewClient(context.Background(),
```
Comment (edwarnicke) [inline test/client_endpoint_test.go:150]: Why is the NSMgr doing retry... I would expect only the NSC to do retry.

Comment (denis-tingaikin) [reply test/client_endpoint_test.go:150]: Oops, this can be removed

Comment (edwarnicke) [reply test/client_endpoint_test.go]: Ah... this is in a test :)

Comment (denis-tingaikin) [reply test/client_endpoint_test.go]: yes, it is a test, and also retry is not needed actually here. It should work without retries.

Comment (denis-tingaikin) [reply test/client_endpoint_test.go]: Forgot to remove debug stuff

**Pull request merged**